megamek/src/megamek/common/options/OptionsConstants.java

```diff
@@ -272,260 +262,251 @@ public class OptionsConstants {
+    public static final String ADVAERORULES_STRATOPS_ECM = "stratops_ecm";  
+    public static final String ADVAERORULES_STRATOPS_SENSOR_SHADOW = "stratops_sensor_shadow";  
+    public static final String ADVAERORULES_STRATOPS_OVER_PENETRATE = "stratops_over_penetrate";  
+    public static final String ADVAERORULES_STRATOPS_SPACE_BOMB = "stratops_space_bomb";  
+    public static final String ADVAERORULES_STRATOPS_BEARINGS_ONLY_LAUNCH = "stratops_bearings_only";  
+    public static final String ADVAERORULES_STRATOPS_BEARINGS_ONLY_VELOCITY = "stratops_bearings_only_velocity";  
+    public static final String ADVAERORULES_STRATOPS_WAYPOINT_LAUNCH = "stratops_waypoint_launch";  
+    public static final String ADVAERORULES_STRATOPS_ADVANCED_SENSORS = "stratops_advanced_sensors";  
+    public static final String ADVAERORULES_VARIABLE_DAMAGE_THRESH = "variable_damage_thresh";  
+    public static final String ADVAERORULES_AT2_NUKES = "at2_nukes";  
+    public static final String ADVAERORULES_AERO_SANITY = "aero_sanity";  
+    public static final String ADVAERORULES_RETURN_FLYOVER = "return_flyover";  
+    public static final String ADVAERORULES_AA_MOVE_MOD = "aa_move_mod";  
+    public static final String ADVAERORULES_ALLOW_LARGE_SQUADRONS = "allow_large_squadrons";  
+    public static final String ADVAERORULES_SINGLE_NO_CAP = "single_no_cap";  
+    public static final String ADVAERORULES_AERO_ARTILLERY_MUNITIONS = "aero_artillery_munitions"; 
+    public static final String ADVAERORULES_EXPANDED_KF_DRIVE_DAMAGE = "expanded_kf_drive_damage";
+    public static final String INIT_INF_MOVE_EVEN = "inf_move_even";  
+    public static final String INIT_INF_DEPLOY_EVEN = "inf_deploy_even";  
+    public static final String INIT_INF_MOVE_LATER = "inf_move_later";  
+    public static final String INIT_INF_MOVE_MULTI = "inf_move_multi";
+    public static final String INIT_PROTOS_MOVE_EVEN = "protos_move_even";  
+    public static final String INIT_PROTOS_DEPLOY_EVEN = "protos_deploy_even";  
+    public static final String INIT_PROTOS_MOVE_LATER = "protos_move_later";  
+    public static final String INIT_PROTOS_MOVE_MULTI = "protos_move_multi";
+    public static final String INIT_INF_PROTO_MOVE_MULTI = "inf_proto_move_multi";
+    public static final String INIT_SIMULTANEOUS_DEPLOYMENT = "simultaneous_deployment";  
+    public static final String INIT_SIMULTANEOUS_MOVEMENT = "simultaneous_movement";  
+    public static final String INIT_SIMULTANEOUS_TARGETING = "simultaneous_targeting";  
+    public static final String INIT_SIMULTANEOUS_FIRING = "simultaneous_firing";  
+    public static final String INIT_SIMULTANEOUS_PHYSICAL = "simultaneous_physical";  
+    public static final String INIT_FRONT_LOAD_INITIATIVE = "front_load_initiative";  
+    public static final String INIT_INITIATIVE_STREAK_COMPENSATION = "initiative_streak_compensation";
+    public static final String RPG_PILOT_ADVANTAGES = "pilot_advantages";  
```

Comment:
Is removing those blank lines between groups reaaaaally an accessibility improvement?

Reply:
Those aren't different groups, they're all game options.